whole-body-controller-core/.gitignore

```diff
@@ -0,0 +1 @@
+/.gradle/
```

Comment:
Auto-generated by eclipse. Should we be adding this?

Reply:
It should be copied from another existing project, Dina knows where to fetch one